charts/redpanda/templates/configmap.yaml

```diff
@@ -351,3 +351,60 @@ data:
 {{- if hasKey .Values.tuning "well_known_io" }}
       well_known_io: {{ .Values.tuning.well_known_io }}
 {{- end }}
+---
+
+{{/* 
+  config.yaml: |
+      clientId: redpanda-console
+      {{ /* if sasl is enabled * }}
+      {{ /* if tls is enabled * }}
+{{- if .Values.listeners.schemaRegistry.enabled }}
+      schemaRegistry:
+        enabled: true
+        urls:
+    {{- if .Values.console.config -}}
+      {{ .Values.console.config | nindent 4 }}
+    {{- end }}
+
+  {{- if .Values.console.roles }}
+  roles.yaml: |
+    roles:
+      {{- tpl (toYaml .Values.console.roles) $ | nindent 6 }}
+  {{- end }}
+
+  {{- if .Values.console.roleBindings }}
+  role-bindings.yaml: |
+    roleBindings:
+      {{- tpl (toYaml .Values.console.roleBindings) $ | nindent 6 }}
+  {{- end }}
+{{- end }}
+*/}}
+---
+# console config
```

Comment:
Is it possible to move this to a helper file and just "get" the configmap from there?

Reply:
Still thinking about placement. This section is more-or-less is a helper file.

Reply:
I'm hesitant because it creates a second layer of indirection where we already have one layer with the configmap file from the subchart.